ci/travis/format.sh

```diff
@@ -5,9 +5,18 @@
 # Cause the script to exit if a single command fails
 set -eo pipefail
 
+FLAKE8_VERSION_REQUIRED="3.7.7"
```

Comment:
Are we also printing the same if `flake8` isn't installed?

Reply:
Not for now. I will also add it.